test/e2e/ml_upgrade_test.go

```diff
@@ -0,0 +1,99 @@
+package e2e
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/stretchr/testify/assert"
+	"github.com/tidwall/gjson"
+	digestAuth "github.com/xinsnake/go-http-digest-auth-client"
+)
+
+func TestHelmMLupgrade(t *testing.T) {
```

Comment:
We already have a upgrade test in upgrade_test.go. Try to merge your test case to that file. Generally we would like fewer e2d test cases to have faster testing time by merge similar test case in the same testing cluster.

Reply:
merged upgrade tests in one file.